src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -95,9 +93,10 @@ private JObject GetInvokeResult(byte[] script, Signers signers = null, bool useD
               }));
             if (useDiagnostic)
             {
+                Diagnostic diagnostic = (Diagnostic)engine.Diagnostic;
```

Comment:
It could be null?

```suggestion
            if (useDiagnostic && engine.Diagnostic is Diagnostic diagnostic )
            {
```

Reply:
We created the object in L80. So it can't be null.